src/test/java/com/exasol/adapter/dialects/exasol/ExasolFromExaQueryRewriterTest.java

```diff
@@ -39,11 +37,13 @@ void beforeEach() {
     void testRewriteWithJdbcConnection() throws AdapterException, SQLException, ExaConnectionAccessException {
```

Comment:
ExaConnectionAccessException is never thrown in this method

Reply:
Eclipse claims it is.